ReactAndroid/src/main/java/com/facebook/react/views/text/ReactBaseTextShadowNode.java

```diff
@@ -291,6 +293,14 @@ public float getEffectiveLineHeight() {
     return useInlineViewHeight ? mHeightOfTallestInlineImage : mLineHeight;
   }
 
+  private static float calculateLetterSpacing(float letterSpacing, int fontSize) {
+    if (Float.isNaN(letterSpacing) || letterSpacing == 0.0f) {
+      return 0.0f;
+    }
+
+    return letterSpacing / fontSize * 2.0f;
```

Comment:
I believe the [approach](https://github.com/facebook/react-native/pull/13877/files#diff-dde94cc0ccc8d662c9efae752698d1c2R82) taken in #13877 is a more correct one with respect to the unit conversions involved; see also my comment on `setFontSize` below.